app/resources/views/pages/home.blade.php

```diff
@@ -33,11 +47,23 @@
 					</a>
 				</div>
 				@endif
+				<!-- HTML FOR SHOWING SEARCH ADDRESS-->
+				<div style="margin-top:10px;">
+							<div class="row">
+										<div class="col-lg-12">
```

Comment:
See if the map still looks good without so many div elements.  I suspect that the <div class="row" and div class="col-lg-12" and corresponding ending tags can be removed without any harm to the layout.

Also, shrink your browser down to see if the map still looks good and a good size at very narrow mobile phone style resolutions.